src/gui/Layout.cpp

```diff
@@ -137,153 +140,131 @@ Rectangle Layout::getVisibleRect()
 	                 gtk_adjustment_get_page_size(scrollHandling->getVertical()));
 }
 
-/**
- * Returns the height of the entire Layout
- */
-double Layout::getLayoutHeight()
+int addif(int lhs, int rhs, bool b)
```

Comment:
Done, but this code snipped is so small, that it should be understandable just by reading the code.
It also is a source-file function so no one can access it from outside the source file.

Reply:
Its used in two functions, i will refactor it to an inline function.
I thought that this function is so trivial, that every compiler will inline it automatically, but godbolt.org convinced me otherwise. The inline keyword works gcc/clang/msvc and the Intel compiler inlining the function.

Reply:
It's inlined when optimizations are turned on (e.g. with `-O1`).